web/admin/src/pages/document/component/ImportDoc/Sitemap.tsx

```diff
@@ -35,6 +36,11 @@ const StepText = {
   },
 };
 
+interface ItemsType extends V1SitemapScrapeResp, V1SitemapParseItem {
```

Comment:
在Sitemap.tsx文件中，类型定义可以进一步明确以提高类型安全性。

```suggestion
// 建议明确指定url属性为非空
interface ItemsType extends V1SitemapScrapeResp, V1SitemapParseItem {
  success: number;
  id: string;
  url: string; // 明确指定url为非空
}